Users/Glob_Patterns.rst

```diff
@@ -13,8 +13,7 @@ Suppose you want ``SpaceConsistencyBear`` to perform an analysis on a file
 .. note::
 
     If you don't know the functions of a bear or how to perform the analysis
-    with a bear, you should go through `Tutorial
-    <http://coala.readthedocs.io/en/latest/Users/Tutorial.html>`_ first.
+    with a bear, you should go through :doc:`Tutorial <Tutorial>`  first.
```

Comment:
```suggestion
    with a bear, you should go through :doc:`Tutorial <Tutorial>` first.
```

Reply:
Also, `the Tutorial` will look better, giving more importance to the word "tutorial"